tests/integration/config.yml

```diff
@@ -0,0 +1,128 @@
+eden:
+    #root directory of eden
+    root: .
+    images:
+        #directory to save images
+        dist: /tmp
+
+        #yml to build docker image
+        docker: /dev/null
+
+        #yml to build vm image
+        vm: /dev/null
+
+    #download eve instead of build
+    download: true
+
+    #eserver is tool for serve images
+    eserver:
+        #port for eserver
+        port: 8888
+
+        #pid for eserver
+        pid: eserver.pid
+
+        #log of eserver
+        log: eserver.log
+
+    #directory to save certs
+    certs-dist: certs
+
+    #directory to save binaries
+    bin-dist: bin
+
+    #ssh-key to put into EVE
+    ssh-key: ../../conf/ssh.key
+
+    #observe logs in tests
+    logs: false
+
+#config is generated by eden
+adam:
+    #location of adam
+    dist: ../../dist/amd64/installer 
+
+    #port of adam
+    port: 6000
+
+    #domain of adam
+    domain: localhost
+
+    #ip of adam for EVE access
+    eve-ip: 127.0.0.1
+
+    #ip of adam for EDEN access
+    ip: 127.0.0.1
+
+    #force adam rebuild
+    force: false
+
+    #certificate for communication with adam
+    ca: ""
+
+eve:
+    #EVE arch (amd64/arm64)
+    arch: amd64
+
+    #EVE os (linux/darwin)
+    os: darwin
+
+    #EVE acceleration (set to false if you have problems with qemu)
+    accel: true
+
+    #variant of hypervisor of EVE (kvm/xen)
+    hv: kvm
+
+    #serial number in SMBIOS
+    serial: 31415926
+
+    #onboarding certificate of EVE to put into adam
+    cert: certs/onboard.cert.pem
+
+    #EVE pid file
+    pid: eve.pid
+
+    #EVE log file
+    log: eve.log
+
+    #EVE firmware
+    firmware: eve/dist/amd64/OVMF.fd
+
+    #eve repo used in clone mode (eden.download = false)
+    repo: https://github.com/lf-edge/eve.git
+
+    #eve tag
+    tag: 5.1.11
+
+    #eve tag for base os
+    base-tag: 5.1.10
+
+    #forward of ports in qemu [(HOST:EVE)]
+    hostfwd:
+        2222: 22
+        5912: 5901
+        5911: 5900
+        8027: 8027
+        8028: 8028
+
+    #location of eve directory
+    dist: eve
+
+    #location of EVE base os directory
+    base-dist: evebaseos
+
+    #file to save qemu config
+    qemu-config: /Users/roman/.eden/qemu.conf
```

Comment:
Would be nice if there weren't romans everywhere like 2000 year ago.
But I assume that is an RFE for eden.

Reply:
LOL -- yeah -- that part I still need to update based on the latest improvements in Eden.